crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -63,7 +63,7 @@ impl PartialEq for SuiTransactionResponse {
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, PartialEq, Eq)]
-#[serde(rename = "TransactionKind")]
+#[serde(rename = "TransactionKind", tag = "type")]
```

Comment:
```suggestion
#[serde(rename = "TransactionKind", tag = "kind")]
```

nit: can we rename it to `kind`?